src/main/scala/symsim/concrete/ConcreteExactRL.scala

```diff
@@ -33,3 +33,24 @@ trait ConcreteExactRL[State, ObservableState, Action]
   /** Convert the matrix Q after training into a Policy map. */
   def qToPolicy (q: Q) (using Ordering[Double]): Policy =
     q.states.map { s => (s, vf.bestAction (q) (s)) }.to (Map)
+
+  def policyEval (policies: List[Policy]): List[List[Double]] =
+    val initials = Randomized.repeat (agent.initialize).take (5)
+    val rL = for
+      s_t <- initials
+    yield policies.map (policy => runEval (policy, s_t, 0.0))
+    rL.toList
+
+  def runEval(policy: Policy, s_t: State, r_acc: Double): Double =
+    if agent.isFinal(s_t) then r_acc
+    else
+      val os_t = agent.observe (s_t)
```

Comment:
this code here belongs to something like `evaluateEpoch` which likely belongs to ExactRL.scala. I would like to separate `runEval`. It requires more thinking so I will write a separate message in the PR in a few minutes.